Chapters/04-http-server.qmd

```diff
@@ -363,8 +363,9 @@ const std = @import("std");
 const Connection = std.net.Server.Connection;
 pub fn read_request(conn: Connection,
                     buffer: []u8) !void {
-    const reader = conn.stream.reader();
-    _ = try reader.read(buffer);
+    var reader = conn.stream.reader(&.{});
+    _ = try reader.getStream().read(buffer);

```

Comment:
`read()` method is deprecated
https://ziglang.org/documentation/master/std/#std.net.Stream.read

Reply:
I have used new IO interface here
https://codeberg.org/Luciogi/zig-http-server-from-scratch/src/branch/main/src/request.zig#L29